constraints/src/main/java/gov/nasa/jpl/aerie/constraints/json/ConstraintParsers.java

```diff
@@ -152,6 +154,22 @@ static JsonParser<Rate> rateF(final JsonParser<Expression<LinearProfile>> linear
               untuple((kind, alias) -> new StartOf(alias)),
               $ -> tuple(Unit.UNIT, $.activityAlias)));
 
+  private static final JsonParser<Duration> durationP =
```

Comment:
:large_orange_diamond: I think private is not needed here?

Reply:
I general I prefer to restrict visibility as much as possible, much like `const`/`final`-correctness. If it can be private, I think it should be.